packages/cardano-graphql-services/src/StakePoolSearch/run.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable import/imports-first */
+require('../../scripts/patchRequire');
+import * as envalid from 'envalid';
+import { DbSyncStakePoolSearchProvider } from './DbSyncStakePoolSearchProvider';
+import { LogLevel, createLogger } from 'bunyan';
+import { Logger } from 'ts-log';
+import { Pool } from 'pg';
+import { StakePoolSearchHttpServer } from './StakePoolSearchHttpServer';
+import { URL } from 'url';
+import { loggerMethodNames } from '../util';
+import onDeath from 'death';
+
+const envSpecs = {
+  API_URL: envalid.url({ default: 'http://localhost:3000' }),
+  DB_CONNECTION_STRING: envalid.str(),
+  LOGGER_MIN_SEVERITY: envalid.str({ choices: loggerMethodNames as string[], default: 'info' })
+};
+
+void (async () => {
+  const env = envalid.cleanEnv(process.env, envSpecs);
+  const apiUrl = new URL(env.API_URL);
+  const connectionString = env.DB_CONNECTION_STRING;
+  const logger: Logger = createLogger({
+    level: env.LOGGER_MIN_SEVERITY as LogLevel,
+    name: 'stake-pool-search-http-server'
+  });
+  const dbPool = new Pool({ connectionString });
+  const stakePoolSearchProvider = new DbSyncStakePoolSearchProvider(dbPool, logger);
+  const server = StakePoolSearchHttpServer.create(
```

Comment:
I think it would be better to be able to use a single server for all providers, where each provider is "mounted" to some path e.g.
```
/stakepoolsearch/
/txsubmit/
```
Then the the http clients can be created using `https://example.com/stakepoolsearch` as root url. Thoughts @rhyslbw ?

Reply:
Yes, but let's do that as a refactor separately @mkazlauskas  to keep this one focused on delivering a working solution based on the current design. I've logged a ticket